utils/vlog/vlog_to_pbtype.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+import yosys_exec
+import lxml.etree as ET
+import argparse, re
+import os, tempfile
+from yosys_json import YosysJson
+
+"""
+Convert a Verilog simulation model to a VPR `pb_type.xml`
+
+The following Verilog attributes are allowed on instances:
+    - (* PB_PATH="../path/to/pb_type.xml" *) : import the pb_type for a given instance
+        from an external XML file instead of including it inline.
+    
+    - (* GENERATE_PB *) : generate (and overwrite if applicable) the above file, as well as including it
+        
+The following are allowed on both instances (except where PB_PATH is set but GENERATE_PB isn't), and
+the top level module.
+    
+    - (* BLIF_MODEL=".latch" *) : specify the corresponding blif_model for a given instance
```

Comment:
I think we should use `CLASS` in both these cases?

My suggestions would look like this I think;
 * `(* TYPE="bel" CLASS="lut" *)` --> `blif_model=".names` and `class="lut"`
 * `(* TYPE="bel" CLASS="routing" *)` --> No `blif_model` and maybe `class="routing"` when I added support for that to `vpr`.
 * `(* TYPE="bel" CLASS="flipflop" *)` --> `blif_model=".latch` and `class="flipflop"`
 * `(* TYPE="blackbox" CLASS="mem" *)` --> `blif_model=".subckt <name of model>` and `class="memory"`
 * `(* TYPE="blackbox" *)` --> `blif_model=".subckt <name of model>"` and no `class=xxx` property?